src/ImageProcessorCore/Colors/ColorTransforms.cs

```diff
@@ -18,9 +19,60 @@ namespace ImageProcessorCore
     public partial struct Color
     {
         /// <summary>
+        /// Blends two colors by lightening.
+        /// </summary>
+        /// <remarks>
+        /// Selects the lighter of the backdrop and source colors.
+        /// The backdrop is replaced with the source where the backdrop is lighter; otherwise, it is left unchanged
+        /// Note: This behavior is on a channel by channel basis, i.e., this rule is applied to each of the 3 RGB color channels separately.
+        /// </remarks>
+        /// <param name="backdrop"></param>
+        /// <param name="source"></param>
+        /// <returns>
+        /// The <see cref="Color"/>.
+        /// </returns>
+        public static Color Lighten(Color backdrop, Color source)
+        {
+            var newVector = new Vector4
```

Comment:
I didn't want to work on alpha because I'm not sure how to handle the pre 
vs post situation. What is the intent of the transforms, to be used for 
internal image processor library operations or to be used by end users? If 
it's internal we can assume all colors are post and provide one 
implementation. If we have to support pre and post then we need to discuss 
how to architect a solution that can be used based on the source colors pre 
or post status.


On February 14, 2016 6:53:28 PM James Jackson-South 
<notifications@github.com> wrote:

>> @@ -18,9 +19,60 @@ namespace ImageProcessorCore
>>      public partial struct Color
>>      {
>>          /// <summary>
>> +        /// Blends two colors by lightening.
>> +        /// </summary>
>> +        /// <remarks>
>> +        /// Selects the lighter of the backdrop and source colors.
>> +        /// The backdrop is replaced with the source where the backdrop is 
>> lighter; otherwise, it is left unchanged
>> +        /// Note: This behavior is on a channel by channel basis, i.e., 
>> this rule is applied to each of the 3 RGB color channels separately.
>> +        /// </remarks>
>> +        /// <param name="backdrop"></param>
>> +        /// <param name="source"></param>
>> +        /// <returns>
>> +        /// The <see cref="Color"/>.
>> +        /// </returns>
>> +        public static Color Lighten(Color backdrop, Color source)
>> +        {
>> +            var newVector = new Vector4
>
> What happens with the alpha channel here?
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/JimBobSquarePants/ImageProcessor/pull/329/files#r52851453